modules/core/src/main/java/org/apache/ignite/configuration/PageReplacementMode.java

```diff
@@ -0,0 +1,85 @@
+/*
+* Licensed to the Apache Software Foundation (ASF) under one or more
+* contributor license agreements.  See the NOTICE file distributed with
+* this work for additional information regarding copyright ownership.
+* The ASF licenses this file to You under the Apache License, Version 2.0
+* (the "License"); you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+package org.apache.ignite.configuration;
+
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * Defines memory page replacement algorithm. A mode is set for a specific {@link DataRegionConfiguration}.
+ */
+public enum PageReplacementMode {
+    /**
+     * Random-LRU algorithm.
+     *
+     * Every time page is accessed, its timestamp gets updated. When a page fault occurs and it's required to replace
+     * some pages, the algorithm randomly chooses 5 pages from the page memory and evicts a page with the latest
+     * timestamp.
+     *
+     * This algorithm have zero maintenance cost, but not very effective in terms of finding next page to replace.
+     * Recommended to use in environments, where there are no page replacements (large enough data region to store all
```

Comment:
How engineer or someone who would read this can detect is there are page replacements at all ? I suggest to add metric link here ?

Reply:
Which metric do you mean? PagesReplaceRate?

Reply:
yes